plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -927,6 +927,7 @@ items:
     name: system:monitoring
   rules:
   - nonResourceURLs:
+    - /flagz
```

Comment:
drop this, the rendered file won't update until the feature is enabled by default

Reply:
missed it, done.